matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/DefaultAuthenticationService.kt

```diff
@@ -406,6 +408,20 @@ internal class DefaultAuthenticationService @Inject constructor(
         )
     }
 
+    override suspend fun isQrLoginSupported(homeServerConnectionConfig: HomeServerConnectionConfig): Boolean {
+        val authAPI = buildAuthAPI(homeServerConnectionConfig)
+        val versions = runCatching {
+            executeRequest(null) {
+                authAPI.versions()
+            }
+        }
+        return if (versions.isSuccess) {
```

Comment:
If there is a network error the client will consider that the feature is not supported (which is maybe acceptable here).

Reply:
I think it is ok to silently hide it on network problems.